content/_data/roadmap/roadmap.yml

```diff
@@ -28,13 +28,14 @@ categories:
     description: Initiatives focused on improving the Jenkins user interface and user experience
     link: /sigs/ux
     initiatives:
-    - name: "Jenkins UI/UX: CSS revamp"
+    - name: "UI/UX: Look and Feel updates"
```

Comment:
Looks better indeed. Not an expert to say if really "Look and Feel" conveys a similar-enough meaning to "Styling" as we discussed off-the-band.
But again, given it's definitely better than saying "CSS" I'll approve this.

Reply:
Will keep it around to facilitate more feedback

Reply:
I like the change

Reply:
Much better indeed